auto_submit/analysis_options.yaml

```diff
@@ -9,4 +9,5 @@ linter:
     constant_identifier_names: false # we have all capitalized enums in check_for_waiting_pull_requests_test.dart
     require_trailing_commas: true
     unawaited_futures: true
-    prefer_final_fields: true
\ No newline at end of file
+    prefer_final_fields: true
```

Comment:
This can be removed as it's in the parent analysis_options.yaml

Reply:
Done.